torch/testing/_internal/common_utils.py

```diff
@@ -1445,8 +1455,8 @@ def make_tensor(size, device: torch.device, dtype: torch.dtype, *,
         high = math.ceil(10 if high is None else min(high, 10))
         return torch.randint(low, high, size, device=device, dtype=dtype)
     elif dtype in floating_types_and(torch.half, torch.bfloat16):
-        low = -9 if low is None else max(low, -9)
-        high = 9 if high is None else min(high, 10)
+        low = (-9 if low is None else max(low, -9)) + domain_eps
```

Comment:
hm, 1e-5 will do nothing in most cases for half and bfloat16

Reply:
That's true. Let's update this to be 1e-3 for float16 and 1e-2 for bfloat16. Both those values will produce deltas around 1, although not at 9. Currently all the domains we have, however, have left or right points of the ranges closer to zero.